### DISCUSSION TITLE
How to enable Jedis Logging in version 3.7.0

### DISCUSSION THREAD

**Discussion by ashishkeshu** [xetorthio/jedis#2664] [Q&A]
Hi Team,

Please help us to understand how can we enable the jedis logging?
I hope there will be different logging levels like error, warning and debug, How we can set by while creating jedis cluster.

Thanks in advance.

**Comment by sazzad16** [comment]
Jedis uses slf4j apis for logging. So if you have any slf4j implementation in your classpath, you'll get the logs (according to the configuration you provide to that implementation).

It is important to note that Jedis does very few logging.

**Reply by gaspardpetit** [reply]
In our case we were interested in monitoring the commands and responses from the server (not only errors, but traces as well) and submitted https://github.com/redis/jedis/pull/2633 for this purpose.
